server/src-lib/Hasura/GraphQL/Transport/HTTP.hs

```diff
@@ -33,6 +33,7 @@ runGQ reqId userInfo reqHdrs req = do
       flip HttpResponse Nothing <$> runHasuraGQ reqId req userInfo resolvedOp
     E.GExPRemote rsi opDef  ->
       E.execRemoteGQ reqId userInfo reqHdrs req rsi opDef
+    E.GExPMixed fieldPlans -> undefined
```

Comment:
This would leave existing functionality as is and only concern with mixed execution tree.